### PR TITLE
docs: Compare front-end api to edge, not proxy

### PR DESCRIPTION
**Pull request by gardleopard**
Proxy is superseeded by edge, lets compare to edge instead of the old proxy.

### CODE REVIEW

**Review by kwasniew — APPROVED** [review]
We could also mention both Edge and Proxy in comparison. But if we're deprecating proxy then Edge is better to stand on its own.

**Review by chriswk — APPROVED** [review]
Looks nice.

**Comment by gardleopard** [comment]
> We could also mention both Edge and Proxy in comparison. But if we're deprecating proxy then Edge is better to stand on its own.

We have that comparison here https://docs.getunleash.io/understanding-unleash/proxy-hosting

File: website/docs/reference/front-end-api.md [hunk 1]
```diff
@@ -8,20 +8,19 @@ The Unleash front-end API was released in Unleash 4.18. You can read more in the
 
 :::
 
-The **Unleash front-end API** offers a simplified workflow for connecting a client-side (front-end) applications to Unleash. It provides the exact same API as the [Unleash proxy](../generated/unleash-proxy.md). The front-end API is a quick and easy way to add Unleash to single-page applications and mobile apps.
+The **Unleash front-end API** offers a simplified workflow for connecting a client-side (front-end) applications to Unleash. It provides the exact same API as [Unleash edge](https://docs.getunleash.io/reference/unleash-edge) and  [Unleash proxy - deprecated](../generated/unleash-proxy.md). The front-end API is a quick and easy way to add Unleash to single-page applications and mobile apps.
```
Comment (thomasheartman) [inline website/docs/reference/front-end-api.md:11]:
```suggestion
The **Unleash front-end API** offers a simplified workflow for connecting a client-side (front-end) applications to Unleash. It provides the exact same API as [Unleash edge](https://docs.getunleash.io/reference/unleash-edge) and  the [Unleash proxy - deprecated](../generated/unleash-proxy.md). The front-end API is a quick and easy way to add Unleash to single-page applications and mobile apps.
```

**Review by thomasheartman — APPROVED** [review]
Nice one! Just a minor suggestion to add a definite article in front of a proxy mention, but all good 😄

**Pull request merged**